docker-compose.yaml

```
@@ -22,6 +24,9 @@ services:
          - 8080
       depends_on:
          - mysql
-
+      networks:
+         - internal-network
 volumes:
    mysql-data:
+networks:
+   internal-network:
```

Só faltou mesmo especificar o driver de rede, de resto está top!
```suggestion
networks:
   internal-network:
      driver: bridge
```